docs/source/upgrade-guide/index.md

```diff
@@ -522,6 +522,145 @@ The `react/jsx-key` rule has been enabled in ESlint for catching missing `key` i
 You might catch some violations in your project or add-on code after running ESlint.
 Adding the missing `key` property whenever the violation is reported will fix it.
 
+### Deprecation notices for Volto 18
+
+#### `@plone/generator-volto`
+
+```{deprecated} Volto 18.0.0
+```
+
+The Node.js-based Volto project boilerplate generator is deprecated from Volto 18 onwards.
+Although you can still migrate your project to Volto 18 using this boilerplate, you should migrate to using [Cookieplone](https://github.com/plone/cookieplone).
+After the release of Volto 18, it will be marked as deprecated, archived, and it won't receive any further updates.
```

Comment:
Reorder to make it clearer that the "after the release" sentence isn't talking about Cookieplone.

```suggestion
After the release of Volto 18, it will be marked as deprecated, archived, and it won't receive any further updates.
Although you can still migrate your project to Volto 18 using this boilerplate, you should migrate to using [Cookieplone](https://github.com/plone/cookieplone).
```